spec/features/users_can_create_projects_spec.rb

```diff
@@ -62,13 +62,17 @@
   end
 
   context "the URN is empty" do
+    before { mock_successful_api_establishment_response(urn: any_args) }
```

Comment:
For me, these feature specs are too much, I would take my  confidence from the model validations specs, we are even checking the error messages, which is great - I like to rely on Rails doing the right thing and showing those messages and not test that with features.

I don't think we need to take this out in this work - but maybe we look to refactor our feature specs soon to set a precedence we want to continue.